src/olympia/amo/tests/__init__.py

```diff
@@ -271,6 +271,9 @@ def generate_api_token(self, user, **payload_overrides):
         from rest_framework_jwt.settings import api_settings
         payload = api_settings.JWT_PAYLOAD_HANDLER(user)
         payload.update(payload_overrides)
+        if ('user_id' in payload_overrides and
+                payload_overrides['user_id'] is None):
```

Comment:
This should never happen and it's definitely weird. I nearly didn't include a test because it shouldn't happen but figured I would